src/Routes/doctor.routes.js

```diff
@@ -0,0 +1,268 @@
+ *                     type: number
+ *                 medicineDescription:
+ *                     type: string
+ *          Hospital:
+ *             type: object
+ *             properties:
+ *                 hospitalName:
+ *                     type: string
+ *                 hospitalAddress:
+ *                     type: object
+ *                     properties:
+ *                         addressLine:
+ *                             type: string 
+ *                         city:
+ *                             type: string
+ *                         postalCode:
+ *                             type: number
+ *                         country:
+ *                             type: string
+ *                 noOfRooms:
+ *                     type: number
+ *                 roomCharge:
+ *                     type: number
+ *          bill:
+ *             type: object
+ *             properties:
+ *                 prescriptionId:
+ *                      type: string
+ *                 doctorCharge:
+ *                      type: number
+ *                 noOfDays:
+ *                      type: number
+ *                 operationCharge:
+ *                      type: number                     
+ *          
+ *             
+ */
+
+/**
+ * @swagger
+ * /doctor/:
+ *  post:
+ *      summary: Registering a new Doctor
+ *      tags:
+ *          - doctors  
+ *      description: Add the new doctor data in database
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Doctor'
+ *      responses:
+ *          200:
+ *              description: Added Successfully
+ *              
+ */ 
+
+doctorRoute.post('/',doctorController.newDoctor)
+
+/**
+ * @swagger
+ * /doctor/hospital:
+ *  post:
+ *      summary: Registering a new hospital
+ *      tags:
+ *          - doctors 
+ *      description: Add the new hospital data in database
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Hospital'
+ *      responses:
+ *          200:
+ *              description: Added Successfully
+ *              
+ */ 
+
+doctorRoute.post('/hospital',verifyToken, doctorController.newHospital)
+
+/**
+ * @swagger
+ * /doctor/medicine:
+ *  post:
+ *      summary: Registering a new Medicine
+ *      tags:
+ *          - doctors 
+ *      description: Add the new medicine data in database
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Medicine'
+ *      responses:
+ *          200:
+ *              description: Added Successfully
+ *              
+ */ 
+
+doctorRoute.post('/medicine',verifyToken,doctorController.newMedicine)
+
```

Comment:
Refered to api practices website ..implemented changes according like here route name changed from verbs to simple words EX : `newMedicine` --> `medicine`